### PR TITLE
Feat: APP-2270 - Input Wallet in Existing tokens flow

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description

- Adds a wrapper for `WalletInput` to hide boilerplate
- Adapts existing tokens flow to use new `WalletInput`
- Adds translation strings for `WalletInput`

Task: [APP-2270](https://aragonassociation.atlassian.net/browse/APP-2270)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2270]: https://aragonassociation.atlassian.net/browse/APP-2270?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/web-app/src/containers/setupCommunity/addExistingToken.tsx [hunk 1]
```diff
@@ -103,20 +112,19 @@ const AddExistingToken: React.FC = () => {
...
                 name={name}
-                // state={error && 'critical'}
+                state={error && 'critical'}
                 value={value}
                 onBlur={onBlur}
                 placeholder={'0x…'}
                 onChange={onChange}
-                // blockExplorerURL={CHAIN_METADATA[network].lookupURL}
+                error={error?.message}
+                blockExplorerURL={tokenAddressBlockExplorerURL}
```
Comment (sepehr2github) [inline packages/web-app/src/containers/setupCommunity/addExistingToken.tsx:123]: You forgot to add token address after Explorer URL

File: packages/web-app/src/utils/constants/chains.ts [hunk 2]
```diff
@@ -156,7 +156,7 @@ export const CHAIN_METADATA: ChainList = {
     name: 'Arbitrum Goerli',
     domain: 'L2 Blockchain',
     logo: 'https://bridge.arbitrum.io/logo.png',
-    explorer: 'https://goerli-rollup-explorer.arbitrum.io/',
+    explorer: 'https://testnet.arbiscan.io/',
```
Comment (sepehr2github) [inline packages/web-app/src/utils/constants/chains.ts:159]: hahaha, Nice Catch

File: packages/web-app/src/containers/setupCommunity/addExistingToken.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/containers/setupCommunity/addExistingToken.tsx:123]: It is added, you might need to rebuild ui-components

**Review by sepehr2github — APPROVED** [review]
Thanks

**Pull request merged**